pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -143,13 +136,6 @@ pub mod pallet {
 		},
 	}
 
-	/// Storage for domain routers.
-	///
-	/// This can only be set by an admin.
-	#[pallet::storage]
-	#[pallet::getter(fn domain_routers)]
-	pub type DomainRouters<T: Config> = StorageMap<_, Blake2_128Concat, Domain, T::Router>;
```

Comment:
I'm happy I have just finished writing the migration 🥲. Though I can imagine I can adapt it rather quickly to the final outcome of all the refactors.

Reply:
Awesome! Thanks William